l2geth/miner/worker.go

```diff
@@ -466,7 +466,7 @@ func (w *worker) l1Tol2StartLoop() {
 		select {
 		case obj := <-w.l1ToL2Sub.Chan():
 			if !w.isRunning() {
-				log.Info("miner receives batchPeriodStartEvent but miner not start")
+				log.Debug("miner receives batchPeriodStartEvent but miner not start")
```

Comment:
Wouldn't it be better to use ERROR?

Reply:
On pending sequencer node, the log will be printed frequently. If use `ERROR`, it will be confusing.

Reply:
👌🏻